packages/grid/helm/syft/templates/backend/backend-service-account.yaml

```diff
@@ -26,7 +26,7 @@ metadata:
     app.kubernetes.io/component: backend
 rules:
   - apiGroups: [""]
-    resources: ["pods", "configmaps", "secrets"]
+    resources: ["pods", "configmaps", "secrets", "services"]
```

Comment:
Looks like we're just performing read-only ops in "KubeUtils.get_service", but this rule will give backend full capabilities over all services. We should move this to the read-only rule below next to `pods/log`

Reply:
Actually we need to expose port on rathole service (on domain) whenever a new rtunnel is setup. Hence we need to patch rathole service. Therefore backend is provided access to both read/write/patch services.
https://github.com/OpenMined/PySyft/blob/766f5250faa1f0b5850c2d4a9974a7d904779024/packages/syft/src/syft/service/network/rathole_config_builder.py#L261

Reply:
ah i see should be a part of kubeutils but we'll do that later.